app/assets/stylesheets/application.scss

```diff
@@ -25,20 +25,20 @@
 
 .nav-tabs {
     align-items: stretch;
-    display: flex;
+    display: flex !important;
```

Comment:
a better approach here would be to use a non-bootstrap class in addition to `.nav-tabs` in the template and define the flex attributes under that instead of using `!important` which can collide with other instances of `!important`

Reply:
Changed! Thanks for the help, @suzidao!